recipes/conversational_voice_agent/load_tts.py

```diff
@@ -0,0 +1,26 @@
+import nltk

```

Comment:
This file seems not to be needed, as exactly the same is done in app.py. 

However, when you add openvino conversion, it should be done in a separate file.